testHelpers/utils.js

```diff
@@ -50,22 +60,32 @@ class Utils {
         qty,
       ],
       {
-        from: seller.address,
         value: txValue,
       }
     );
 
-    return returnTx ? txOrder : txOrder.logs[0].args._tokenIdSupply.toString();
+    const txReceipt = await txOrder.wait();
+    let eventArgs;
+
+    events.assertEventEmitted(
```

Comment:
Good idea to add this assertion here.

Reply:
I am not sure if i understood correctly. Assuming you'd like to implement some assertions for the event that was emitted on the particular function - we had a lot of tests covering this scenario in the actual unit testing. This is only a utility file and this function is being used only to  get the arguments from the emitted event. If this is somehow misleading i can implement a new function instead.

Reply:
There was no event checking in the preview version of the utility.js file. I was just commenting that it was good that it was added.